tests/ps_schedstatistics/tests/01-run.py

```diff
@@ -40,8 +41,10 @@ def _check_startup(child):
 
 def _check_help(child):
     child.sendline('')
-    child.expect('>')
+    child.expect('> ')
```

Comment:
Instead of adding sleep can you just `child.expect('> ', timeout=0.1)`?

Reply:
No, that would make pexpect give '> ' more time, right? I intended to add a slight delay after receiving the prompt, before sending the next line.